ui/admin/app/components/form/field/list-wrapper/select/index.hbs

```diff
@@ -0,0 +1,79 @@
+<Hds::Table class='list-wrapper-field' name={{@name}}>
+
+  <:body as |B|>
+
+    {{#each this.options as |select index|}}
+      <B.Tr>
+        <B.Td>
+          <Hds::Form::Select::Field
+            @value={{select.value}}
+            @width={{@width}}
+            disabled={{@disabled}}
+            aria-label={{t 'titles.value'}}
+            {{on 'change' (set-from-event select 'value')}}
+            as |F|
+          >
+            <F.Options>
+              {{#each @options as |selectOption|}}
```

Comment:
We should call the argument something besides `@options` since that's normally the existing input values.

Reply:
@ZedLi changing this `@selectOptions`